pkg/sdk/helper.go

```diff
@@ -88,12 +98,12 @@ func (h *SDKHelper) WithSDKVersion(version string) error {
 }
 
 // WithAPIVersion sets the `apiVersion` field.
-func (h *SDKHelper) WithAPIVersion(apiVersion string) {
+func (h *Helper) WithAPIVersion(apiVersion string) {
 	h.apiVersion = apiVersion
 }
 
 // API returns the aws-sdk-go API model for a supplied service model name.
-func (h *SDKHelper) API(serviceModelName string) (*SDKAPI, error) {
+func (h *Helper) API(serviceModelName string) (*SDKAPI, error) {
```

Comment:
I know you didn't change any logic here, but why have the helper do this instead of a constructor for `SDKAPI` requiring serviceModelName?

Reply:
I don't have any history for this method - they predate my introduction to the project. My best guess is that we don't care about the service name anywhere else in the `Helper`, but I think over time we have expanded it and now we have an awkward waterfall of passing it between methods.